subgraphs/across-v2/src/handlers/liquidity.ts

```diff
@@ -0,0 +1,72 @@
+import { Bytes } from "@graphprotocol/graph-ts";
+import {
+  LiquidityAdded,
+  LiquidityRemoved,
+} from "../../generated/HubPool/HubPool";
+import { SDK } from "../sdk/protocols/bridge";
+import { BridgeConfig } from "../sdk/protocols/bridge/config";
+import {
+  BridgePermissionType,
+  BridgePoolType,
+} from "../sdk/protocols/bridge/enums";
+import { Versions } from "../versions";
+import {
+  ACROSS_HUB_POOL_CONTRACT,
+  ACROSS_PROTOCOL_NAME,
+  Pricer,
+  TokenInit,
+} from "../util";
+
+const conf = new BridgeConfig(
```

Comment:
These contract address constants will be different across different networks, we need a way to select that

Reply:
The liquidity is managed on the hub pool (deployed only on the mainnet). This will be constant for that reason. However, the bridge config will have different id for the transfers as done here https://github.com/fortysevenlabs/messari-subgraphs/blob/across-v2/subgraphs/across-v2/src/handlers/transfers.ts#L34